lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -931,6 +913,9 @@ export class UserAgentApplication {
 
       // open the window
       const popupWindow = window.open(urlNavigate, title, "width=" + popUpWidth + ", height=" + popUpHeight + ", top=" + top + ", left=" + left);
+      if (!popupWindow || popupWindow == null) {
+        throw ClientAuthError.createPopupWindowError();
```

Comment:
I wonder if this is a client auth error or something that is a problem with our lib.  If A pop up window cant be created thats a big red flag right>?

Reply:
Window.open returns null if the window isn't opened. I haven't found a way to get more information about why it wasn't opened, but we should throw an error if we can't open one.